### PR TITLE
Update documentation for running the broker

### PR DESCRIPTION
**Pull request by sarneaud**

### CODE REVIEW

**Comment by afeld** [comment]
Awesome, thank you! @jmcarp Mind confirming?

File: README.md [hunk 1]
```diff
@@ -36,18 +40,21 @@ There are important environment variables that should be overriden inside the `m
 Catalog.yaml contains a list of service(s) offered with plans. It contains no secrets.
 Prior to pushing, complete the catalog.yaml for your environment. It is architected where the service name (e.g. rds) is the mapping between it and the service details.
 
-#### Secrets.yml
+If you aren't sure what's required, see the respective service user guides:
+* [RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_GettingStarted.html)
+* [S3](https://docs.aws.amazon.com/AmazonS3/latest/gsg/GetStartedWithS3.html)
```
Comment (jmcarp) [inline README.md]: Just curious--why link to the S3 docs here? S3 isn't currently supported by this broker, but linking might imply that it is.

File: README.md [hunk 2]
```diff
@@ -36,18 +40,21 @@ There are important environment variables that should be overriden inside the `m
...
 
-secrets.yml contains the all of the secrets for the different resources. 
+#### Secrets.yml
 
+secrets.yml contains all of the secrets for the different resources. Each plan requires a database (not necessarily hosted on AWS).
```
Comment (jmcarp) [inline README.md]: To be specific, each plan of type rds requires a database.

File: README.md (hunk 1 above)
Comment (sarneaud) [inline README.md]: S3 is mentioned elsewhere in the docs, so I just added the link for consistency.  Do you want this link removed?

File: README.md (hunk 2 above)
Comment (sarneaud) [inline README.md]: I'm a bit confused here.  RDS is the only type implemented now, but do you think that future services will be implemented differently?

**Pull request closed without merging**